BEIMA.DB.Schemas/device-schema.txt

```diff
@@ -0,0 +1,25 @@
+{
+    "templateId": String,
+    "serialNumber": String,
+    "deviceTag": String,
+    "manufacturer": String,
+    "modelNum": String,
+    "serialNum": String,
+    "yearManufactured": int,
+    "notes": String,
+    "location": {
+        "buildingId": String,
+        "notes": String,
+        "latitude": String,
+        "longitude": String
+    },
+    "lastModified": {
+        "date": Date,
+        "user": String
```

Comment:
Should we add in a user table? Maybe every time we have a login from a new user, we could add it into the user table. Might make for better tracking and could make it easier for use to assign certain access to specific users later on.

Reply:
I think this will come later when we start work on the authentication part of BEIMA

Reply:
Yes, this is the ultimate intention, but not part of the scope of this task/PR. We will have a collection named "users", and there will be a record for each user containing login information and other information about the user.

This "user" field will store the username of the user who last modified the device.